packages/compass-e2e-tests/tests/connection.test.ts

```diff
@@ -88,10 +93,10 @@ describe('SRV connectivity', function () {
         undefined,
         'either'
       );
+      screenshot = undefined;
     } finally {
-      await disconnect(browser);
```

Comment:
No need to disconnect if we're going to close the app and disconnecting in the finally means the screen changes before we can take the screenshot.